server/sched.go

```diff
@@ -201,7 +201,7 @@ func (s *Scheduler) processPending(ctx context.Context) {
 					}
 
 					// Load model for fitting
-					ggml, err := llm.LoadModel(pending.model.ModelPath, 0)
+					ggml, err := llm.LoadModel(pending.model.ModelPath, 1024)
```

Comment:
@mxyng I think we missed this one in #10382. Is there anything else this would break? I'm guessing we didn't previously use smaller arrays as part of estimates before my PR, but wanted to check because the prerelease is missing this